pkg/services/object/tombstone/verify.go

```diff
@@ -130,28 +132,26 @@ func (v *Verifier) verifyMember(ctx context.Context, cnr cid.ID, member oid.ID)
 
 func (v *Verifier) verifyV1Child(ctx context.Context, cnr cid.ID, sID object.SplitID) error {
 	filters := object.SearchFilters{}
+	filters.AddPayloadSizeFilter(object.MatchNumGT, 0)
```

Comment:
1. The first filter should be the most restrictive one, in this case it'd be split ID filter.
2. `> 0` is wrong, we need `== 0` and in fact we can limit the count to 1 as well then, v1 link objects have no payload and we want link objects here.